charts/csi-secrets-store-provider-azure/README.md

```diff
@@ -31,10 +31,10 @@ The following table lists the configurable parameters of the csi-secrets-store-p
 | `image.pullPolicy` | Image pull policy | `IfNotPresent` |
 | `image.tag` | Azure Keyvault Provider image | `0.0.6` |
 | `linux.enabled` | Install azure keyvault provider on linux nodes | true |
-| `linux.nodeSelector` | Node Selector for the daemonset on linux nodes | `beta.kubernetes.io/os: linux` |
+| `linux.nodeSelector` | Node Selector for the daemonset on linux nodes | `kubernetes.io/os: linux` |
```

Comment:
these values are not used in the chart

Reply:
The values are actually used in the chart - https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/charts/csi-secrets-store-provider-azure/templates/provider-azure-installer.yaml#L34-L37, but I agree we can do the same as what we did for the driver to ensure mandatory labels exist.